app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/fragments/DialogPopupFragment.kt

```diff
@@ -1,27 +1,22 @@
-package edu.uc.hickmadc.customnotifications.ui.main
+package edu.uc.hickmadc.customnotifications.ui.main.fragments
 
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
 import androidx.fragment.app.DialogFragment
-import androidx.fragment.app.FragmentContainer
 import edu.uc.hickmadc.customnotifications.R
-import java.io.FileDescriptor
-import java.io.PrintWriter
-import java.util.zip.Inflater
 
 
-
-class dialogfragment: DialogFragment() {
+class DialogPopupFragment: DialogFragment() {
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
     ): View?{
         //gets dialog fragment
         var rootView: View= inflater.inflate(R.layout.notification_dialog,container,false)
         //gets ids for buttons
-        var btnClose= rootView.findViewById<Button>(R.id.closeButton)
-        var btnadd = rootView.findViewById<Button>(R.id.btnAdd)
+        var btnClose= rootView.findViewById<Button>(R.id.btnClose)
+        var btnConfirm = rootView.findViewById<Button>(R.id.btnConfirm)
```

Comment:
Suggestion 3: I renamed some of the variables in the dialogfragment class as well as the notification_dialog xml file to better fit the project's naming standards, and I also changed the "btnadd" to "btnConfirm" - I believe that's what the team was wanting to connect to on line 19, but there is a "btnAdd" also referenced in the Main Activity XML file. Since the variable isn't in use yet, I'm not sure whether they intend to implement btnConfirm or btnAdd.
Source: https://kotlinlang.org/docs/coding-conventions.html#choose-good-names

Reply:
Good catch.   Uses proper naming conventions.  More readable, too.